File: ts-sdk/src/sdk/models/operations/getcurrentuser.ts
import { SpeakeasyMetadata, SpeakeasyBase } from "../../../internal/utils";



export class GetCurrentUserResponse extends SpeakeasyBase {
  @SpeakeasyMetadata()
  contentType: string;

  @SpeakeasyMetadata()
  statusCode: number;

  @SpeakeasyMetadata()
  getCurrentUser200ApplicationJSONObject?: Record<string, Record<string, any>>;
}

File: python-sdk/src/planetscaleapi/models/operations/get_current_user.py
import dataclasses
from typing import Any,Optional


@dataclasses.dataclass
class GetCurrentUserResponse:
    content_type: str = dataclasses.field()
    status_code: int = dataclasses.field()
    get_current_user_200_application_json_object: Optional[dict[str, dict[str, Any]]] = dataclasses.field(default=None)
    
